src/style.css

```diff
@@ -1709,4 +1710,36 @@ input.tAUuQ {
 
 .OmA97e {
   z-index: 0 !important;
+}
+
+/* Header Tag FIX*/
+.fOvfyc .B7SYid{
+  color: #ffffff;
+}
+
+/* Mark As Done/Unsubmit Tab*/
+.qyKend{
+  color:#ffffff;
+}
+
+.VcNzjd{
+  color:#ffffff !important;
+}
```

Comment:
I was unable to find what bug this fixes. This should be removed or a screenshot should be given.

Reply:
**This Bug is present in google chrome/edge/opera**

Submit Confirmation / Mark As Done
![image](https://github.com/user-attachments/assets/9a28faad-0b12-4350-af3a-ccdd05032678)

**Unsubmit Confirmation**
![image](https://github.com/user-attachments/assets/e9f197a9-5c20-4fcf-8d42-54ef3d3de9ce)

Reply:
**This changes the text to white and the code block**

`.cC1eCc .VfPpkd-P5QLlc{
  background-color: #454545 !important;
  border-width: 0 !important;
  box-shadow: 0 1px 3px 0 rgb(60 64 67 / 0%), 0 4px 8px 3px rgb(60 64 67 / 0%)!important;
}`

**changes the background to #454545.**

Reply:
```css
/* Header Tag FIX*/
.fOvfyc .B7SYid{
  color: #ffffff;
}

/* Mark As Done/Unsubmit Tab*/
.qyKend{
  color:#ffffff;
}

.VcNzjd{
  color:#ffffff !important;
}
```

I'm trying to find what setting the text color white does since removing this code didn't change anything during my testing. Seem unnecessary.